File: sentimental-mario-less/mario.py
import cs50


while True:
    answer = cs50.get_int("Height: ")
    if answer > 0 and answer < 9:
        break

row = 0
column = 0
for row in range(answer):
    for column in range(answer):
        if row + column < answer - 1:
            print(" ", end="")
        else:
            print("#", end="")
        column += 1

    print()
    row += 1

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int bricks);

int main(void)
{
    // Prompt the user for the pyramid's height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);

    // Print a pyramid of that height
    for (int i = 0; i < n; i++)
    {
        // Print row of bricks
        print_row(n - i - 1, i + 1);
    }
}

void print_row(int spaces, int bricks)
{
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }

    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}
